src/views/CloneRepo.vue

```diff
@@ -1,20 +1,66 @@
 <template>
   <ViewWrapper title="Clone down a repository">
-    <div>
+    <label for="repo-url-input">
+      <strong>
+        Enter url:
+      </strong>
+    </label>
+    <div class="repo-url-select">
+      <input v-model="url" class="repo-url-input" placeholder="Enter url" />
```

Comment:
This needs an ID for the label to be accessible for screen readers
```suggestion
      <input v-model="url" class="repo-url-input" placeholder="Enter url" id="repo-url-input" />
```